### DISCUSSION TITLE
Prediksi Image CNN

### DISCUSSION THREAD

**Discussion by MbakGrace** [teamalgoritma/community#837] [Q&A] [answered]
Dear Team Algoritma,
Mohon izin bertanya terkait prediksi gambar yang saya lakukan mengapa gambarnya tidak dapat diproses ya ? muncul error seperti ini
<img width="608" alt="code-error-1" src="https://github.com/teamalgoritma/community/assets/154303680/d5d24400-cc31-4cc9-9f9f-4db3362e7de1">

setelah saya ubah shapenya muncul keterangan error yang sama

<img width="619" alt="code-error-2" src="https://github.com/teamalgoritma/community/assets/154303680/fbc4a5f9-a516-41d7-861d-8f1b47498dc6">

**Comment by finesaaa** [comment, marked as answer]
Sebaiknya bu Grace lakukan transpose telebih dahulu agar tidak terjadi error tersebut Bu
dengan cara berikut transposed_array = normalized_image_array.transpose(1, 0, 2)
